orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -414,6 +415,8 @@ def _get_bert_for_sequence_classification_model(
         config=config,
     ).to(device)
 
+    if is_training:
+        model.train()
```

Comment:
```suggestion
        model.train()
    else:
        model.eval()
```
To covert the case where `is_training is False and `model.is_training is True`

Reply:
done. thx